src/components/MainApp.tsx

```diff
@@ -43,6 +44,7 @@ const MainApp = ({ children }) => {
             <Sidebar />
             <Layout style={{ background: colors.white }}>
                 <Header />
+                <DonationBanner />
```

Comment:
chore: before this is ready it needs to have a feature flag

Reply:
This is the first time I've done this, if you can confirm if this is correct - @pepermao @caneppelevitor @thesocialdev 

commit:
13d1b29427d4e11e9cf502b8de2eea065fee6604